### PR TITLE
blanks are special again

### PR DESCRIPTION
**Pull request by wasade**
Yet another way to describe body sites for blanks

### CODE REVIEW

**Comment by wasade** [comment]
@squirrelo @EmbrietteH, is this case still necessary to handle?

**Comment by squirrelo** [comment]
Still investigating.

**Comment by wasade** [comment]
The changes just pushed in were necessary to resolve issues encountered with the production data.

**Comment by wasade** [comment]
Assuming tests pass, would it be possible for a rapid review on this? 

cc @squirrelo @EmbrietteH @ElDeveloper @josenavas @antgonza

**Comment by wasade** [comment]
Tests _should_ be passing, possible for review please?

**Comment by ElDeveloper** [comment]
No, stop asking ... jk, I'm on it.

File: ipynb/primary-processing/04-prepare_metaanalyses.md [hunk 1]
```diff
@@ -40,70 +40,76 @@ We're also going to generate some new files, so let's get them setup.
 >>> ag_pgp_hmp_gg_cleaned_md = agu.get_new_path(agenv.paths['ag-pgp-hmp-gg-cleaned-md'])
 ```
 
-The first step in the process is to merge the the individual tables into larger ones, and then to merge the larger tables into a final one. We're not using QIIME's `parallel_merge_otu_tables.py` here as we also need one of the intermediate tables for subsequent processing.
-
-The first merge we'll do is between the Global Gut and the American Gut.
+We also need to make sure the metadata (the information about the samples) are also merged and consistent. Prior to merge, we're going to add in some additional detail about every sample, such as a column in the mapping file that is the combination of the study title and the body site. We're also going to "generalize" body sites to the type of site they're from (e.g., the back of the hand is just "skin"). This process will also clean the metadata to remove blanks and unknown sample types.
```
Comment (ElDeveloper) [inline ipynb/primary-processing/04-prepare_metaanalyses.md]: Very minor, but the usage of the term mapping file comes undefined in this paragraph, and it seems like you were just introducing to the concept of "metadata".

File: scripts/mod2_pcoa.py [hunk 2]
```diff
@@ -243,17 +242,18 @@ def country(coords, mapping_file, output, filename, sample, distmat):
...
     sub_coords = c_df[mf.COUNTRY == grp]
     plt.scatter(sub_coords[0], sub_coords[1], color=color,
                 edgecolor=np.asarray(color)*0.6, lw=LINE_WIDTH,
                 alpha=ALPHA)
 
     # plot participant's dot
-    plt.scatter(c_df.loc[sample][0], c_df.loc[sample][1],
-                color=grp_colors[mf.loc[sample_to_plot]['COUNTRY']],
+    plt.scatter(c_df.loc[sample_to_plot][0], c_df.loc[sample_to_plot][1],
+                color=color_Highlight,
```
Comment (ElDeveloper) [inline scripts/mod2_pcoa.py]: Any reason why highlight should be capitalized in the variable name?

**Comment by ElDeveloper** [comment]
Looks good to me, just a copule of comments, nothing blocking. Should be ready to go granted that tests pass.

**Comment by mortonjt** [comment]
:+1:

File: scripts/mod2_pcoa.py (hunk 2 above)
Comment (wasade) [inline scripts/mod2_pcoa.py]: Not sure, didn't you write this code? :smile:

File: ipynb/primary-processing/04-prepare_metaanalyses.md (hunk 1 above)
Comment (wasade) [inline ipynb/primary-processing/04-prepare_metaanalyses.md]: @mortonjt is going to clean this up

Comment (ElDeveloper) [inline ipynb/primary-processing/04-prepare_metaanalyses.md]: Sounds good!

**Pull request merged**